### PR TITLE
Refactored Chapter 8 How to deal with timestamps.py

### PR DESCRIPTION
**Pull request by ps2016-mt**

### CODE REVIEW

File: cookbook/Chapter 8 - How to deal with timestamps.py [hunk 1]
```diff
@@ -1,50 +1,106 @@
...
 
 # Read it, and remove the last row
-popcon = pd.read_csv(
-    "../data/popularity-contest",
-    sep=" ",
-)[:-1]
-popcon.columns = ["atime", "ctime", "package-name", "mru-program", "tag"]
-popcon[:5]
+# Step 1: Read the text file line by line (plain text handling)
+file_path = r"C:/Users/psingh/polars_update/data/popularity-contest"
```
Comment (yuanjingshi) [inline cookbook/Chapter 8 - How to deal with timestamps.py]: File path contains username that is local and will not be able to run on other pcs. Consider use relative path.

File: cookbook/Chapter 8 - How to deal with timestamps.py [hunk 2]
```diff
@@ -1,50 +1,106 @@
...
+# Step 3: Sort by 'ctime' in descending order and display the top 10 rows
+nonlibraries_sorted = nonlibraries.sort("ctime", descending=True).head(10)
 
-# Now we can use pandas' magical string abilities to just look at rows where the package name doesn't contain 'lib'.
-nonlibraries = popcon[~popcon["package-name"].str.contains("lib")]
-nonlibraries.sort_values("ctime", ascending=False)[:10]
+# Display the sorted, filtered data
+print(nonlibraries_sorted)
 
 # TODO: please reimplement this using Polars
```
Comment (yuanjingshi) [inline cookbook/Chapter 8 - How to deal with timestamps.py]: can remove this comment.

File: cookbook/Chapter 8 - How to deal with timestamps.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 # Read it, and remove the last row
 # Step 1: Read the text file line by line (plain text handling)
-file_path = r"C:/Users/psingh/polars_update/data/popularity-contest"
+file_path = r"file_path"
```
Comment (yuanjingshi) [inline cookbook/Chapter 8 - How to deal with timestamps.py]: data/popularity-contest.csv

**Review by yuanjingshi — COMMENTED** [review]
Suggest a change

File: cookbook/Chapter 8 - How to deal with timestamps.py [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
+import os
 
 # %%
 # Parsing Unix timestamps
 # It's not obvious how to deal with Unix timestamps in pandas -- it took me quite a while to figure this out. The file we're using here is a popularity-contest file of packages.
 
 # Read it, and remove the last row
-# Step 1: Read the text file line by line (plain text handling)
-file_path = "../data/gdp_data.csv"
+# Assuming 'data/gdp_data.csv' is in a 'data' folder within the current working directory
```
Comment (yuanjingshi) [inline cookbook/Chapter 8 - How to deal with timestamps.py:11]: Nice!

**Review by yuanjingshi — APPROVED** [review]

**Comment by ty338jy** [comment]
Well done guys! Merged!

**Pull request merged**